docs/index.md

```diff
@@ -55,6 +55,25 @@ Released in April 2021, ONNX Runtime Training provides a one-line addition for e
 
 Using the ORTModule class wrapper, ONNX Runtime for PyTorch runs the forward and backward passes of the training script using an optimized automatically-exported ONNX computation graph. ORT Training uses the same graph optimizations as ORT Inferencing, allowing for model training acceleration. 
 
+The ORTModule is instantiated from [`torch-ort`](https://github.com/pytorch/ort) backend in PyTorch. This new interface enables a seamless integration for ONNX Runtime training in a PyTorch training code with minimal changes to the existing code.
+
+### Install ONNX Runtime Training package
```

Comment:
There is already an install section here for ORT Training: https://www.onnxruntime.ai/docs/how-to/install.html#training

On the homepage there is a "Get started with ORT for training" link - should this link be updated to point to https://www.onnxruntime.ai/docs/tutorials/training/ort-training.html? Currently it goes to the parent of this page, which just contains a link to the ort-training.html page. (one unnecessary hop)

Reply:
The table only shows the onnxruntime-training packages. Training is installed via `torch-ort`. We need the basic install steps at a high level somewhere.

Yes, the link to Get started with ORT for Training should be updated.

Reply:
Yes, understood that the basic install steps should be documented. What I'm suggesting is adding the install instructions in https://www.onnxruntime.ai/docs/how-to/install.html#training rather than creating a new install section on this page to keep things in one place.

Can you update the Get Started with ORT for Training link as part of this PR? What do you suggest it should link to? Ideally this links to a page with a get-started style tutorial, but currently I think the closest to that lives here: https://github.com/microsoft/onnxruntime-training-examples/tree/master/huggingface 
CC @natke as FYI

Reply:
Updated based on offline chat.